tests/testAutopas/addNameSpace.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
add some context for our future selfs what this script can / should be used for

Reply:
also please mention somewhere that all tests should have their own namespace (probably contribution guidelines)

Reply:
done